engine/src/lnxdcs.cpp

```diff
@@ -1375,30 +1375,33 @@ void MCScreenDC::enablebackdrop(bool p_hard)
 {
 	bool t_error;
 	t_error = false;
-	
+
 	if (p_hard && backdrop_hard)
 		return;
-		
+
 	if (!p_hard && backdrop_active)
 		return;
-	
+
 	if (p_hard)
 		backdrop_hard = true;
 	else
 		backdrop_active = True;
-	
+
 	t_error = (backdrop == DNULL) ;
-	
-	if (!t_error)	
+
+	if (!t_error)
 	{
-        MCstacks -> refresh();
+        // MDW-2018-04-21 [[ bugfix_17323 ]]
+        //MCstacks -> refresh();
         gdk_window_set_functions(backdrop, GdkWMFunction(0));
         gdk_window_set_decorations(backdrop, GdkWMDecoration(0));
         gdk_window_set_skip_taskbar_hint(backdrop, TRUE);
         gdk_window_set_skip_pager_hint(backdrop, TRUE);
 	gdk_window_move_resize(backdrop, 0, 0, device_getwidth(), device_getheight());
         gdk_window_lower(backdrop);
 	gdk_window_show_unraised(backdrop);
+    // MDW - refresh *after* the gdk calls
```

Comment:
Here's the one-line change.

Reply:
@mwieder great! We could probably just move it after the else block and delete the call to refresh there.